### DISCUSSION TITLE
Weather API Queries

### DISCUSSION THREAD

**Discussion by vinolys** [weather-gov/api#742] [General]
In the weather forecast API "https://api.weather.gov/gridpoints/{gridID}/{gridX},{gridY}/forecast",  do grid ID, grid X and grid Y values remain the same for a particular latitude and longitude values? Can I make the query once and save the grid ID, grid X and grid Y values and use it for subsequent queries?

**Comment by sullynole** [comment]
This is addressed in the Examples tab here: https://www.weather.gov/documentation/services-web-api

> Applications may cache the grid for a location to improve latency and reduce the additional lookup request; however, it is important to note that while it generally does not occur often, the gridX and gridY values (and even the office) for a given coordinate may occasionally change. For this reason, it is necessary to check back to the /points endpoint periodically for the latest office/grid mapping.

**Reply by vinolys** [reply]
Thanks!
